web/source/kmwcallback.js

```diff
@@ -346,16 +348,13 @@
       retVal = (keyCode == Lrulekey);         // I3318, I3555
     }
     if(!retVal) this._DeadkeyResetMatched();  // I3318
-    return retVal;                            // I3318
+    return retVal != 0;                            // I3318
```

Comment:
We need to check how this is being used in keyboards before we publish. Because if the keyboards test against `0` instead of `false` and use strict equality, then we have to fix the type information in the comment rather than the return value!